### DISCUSSION TITLE
Support for MYPINT (Malaysia Peppol)

### DISCUSSION THREAD

**Discussion by billyljren** [OxalisCommunity/oxalis#663] [Q&A]
Hi, I would like to know where to start making updates to enable support for MYPINT format. Any pointer / guidance is much appreciated.

Thank you

**Comment by aaron-kumar** [comment]
@billyljren : If you are paying Oxalis member and will be joining "Oxalis Community Technical Dialog Meeting" on 6th March 2024 then you will receive in-depth knowledge as how to achieve this.
